### PR TITLE
Dev freertos

### PR DESCRIPTION
**Pull request by e-puerto**
Update version / Fix clone issue.

### CODE REVIEW

**Comment by bsousi5** [comment]
Add Kevin and Carsten into the loop since FreeRTOS support also ended in FS.

File: wit-manifest.json [hunk 1]
```diff
@@ -174,4 +174,24 @@
         "name": "example-i2c",
         "source": "git@github.com:sifive/example-i2c.git"
     }
```
Comment (nategraff-sifive) [inline wit-manifest.json]: You just need some commas after these close braces to make it valid JSON.

**Comment by cgsfv** [comment]
I tried to create a project in Freedom Studio from example-freertos-blinky on my Mac, without success.

A few observations:
- SystemView.mk and FreeRTOS.mk are using realpath, which is not available on Mac, so that needs to be rewritten to avoid that
- In Freedom Studio we are create standalone projects from freedom-e-sdk, and for freertos this gives a "make[1]: *** No rule to make target '../../scripts/eclipse.mk'.  Stop." error

**Comment by e-puerto** [comment]
Hi Carsten,
for the point 1 - I will check how i can modify it
for the point 2, nate report it, so i push modification this morning in
order to fix it ;) if you update the branch it should be fixed.

Regards,
Emmanuel.


*Emmanuel Puerto*
Application Engineer, Platform Security
*SiFive France*
E: emmanuel.puerto@sifive.com
P: +33 681-359-346 <+33%20681-359-346>
W: www.sifive.com

**Comment by e-puerto** [comment]
Hi Carsten,
I just re-test a fresh install by doing: 
> git clone --recursive https://github.com/sifive/freedom-e-sdk.git
> cd freedom-e-sdk
> git checkout dev_freertos 
> git submodule update --init --recursive
> make PROGRAM=example-freertos-blinky TARGET=sifive-hifive1-revb CONFIGURATION=debug software

And the elf is generated correctly.

**Comment by cgsfv** [comment]
Freedom Studio uses make standalone instead of make software:

```
make standalone STANDALONE_DEST=/eclipse/fs-runtime/qemu-sifive-e31-example-freertos-blinky TARGET=qemu-sifive-e31 PROGRAM=example-freertos-blinky RISCV_PATH=/dev-root/SiFive/riscv64-unknown-elf-gcc-8.3.0-2019.08.0
cd /eclipse/fs-runtime/qemu-sifive-e31-example-freertos-blinky
make all CONFIGURATION=debug
```

Which gives the other result ;-)

**Comment by e-puerto** [comment]
I see, the standalone target is not yet made to take in count the
thirdparty directory. neither additional mk file from script directory. I
will try to modify it.

*Emmanuel Puerto*
Application Engineer, Platform Security
*SiFive France*
E: emmanuel.puerto@sifive.com
P: +33 681-359-346 <+33%20681-359-346>
W: www.sifive.com

**Pull request closed without merging**